Aquality.WinAppDriver/src/Aquality.WinAppDriver/Actions/KeyboardActions.cs

```diff
@@ -26,10 +26,25 @@ public void ReleaseKey(ModifierKey keyToRelease)
             PerformAction(actions => actions.KeyUp(keyToRelease.GetKeysString()));
         }
 
-        public void SendKeys(string keySequence)
+        public void SendKeys(string keySequence, ActionKey? sendAfterSequence = null)
         {
-            LogAction("loc.keyboard.sendkeys", keySequence);
-            PerformAction(actions => actions.SendKeys(keySequence));
+            var valueToLog = $"{keySequence}{(sendAfterSequence == null ? string.Empty : $" + {sendAfterSequence}")}";
+            LogAction("loc.keyboard.sendkeys", valueToLog);
+            var valueToSend = $"{keySequence}{sendAfterSequence.GetKeysOrEmptyString()}";
+            PerformAction(actions => actions.SendKeys(valueToSend));
+        }
+
+        public void SendKeys(ActionKey key, int times = 1)
+        {
+            if (times == 1)
+            {
+                LogAction("loc.keyboard.sendkey", key);
+            }
+            else
+            {
+                LogAction("loc.keyboard.sendkey.times", key, times);
+            }
```

Comment:
lets move this if\esle to some private method or internal extension, because it is duplicated 2 times.